api/services/control/control.proto

```diff
@@ -18,6 +18,7 @@ service Control {
 	rpc Status(StatusRequest) returns (stream StatusResponse);
 	rpc Session(stream BytesMessage) returns (stream BytesMessage);
 	rpc ListWorkers(ListWorkersRequest) returns (ListWorkersResponse);
+	rpc Version(VersionRequest) returns (VersionResponse);
```

Comment:
Might be better to do something like `Info()` (eg. see comment) so that when we need more fields we don't need to make another method for them.

Reply:
Yes `Info()` sounds better. Done.